mitodl/mitxonline
# Removed unneeded auth fields

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #29 

#### What's this PR do?
(Required)

#### How should this be manually tested?
- Create a new account. Ensure that you can successfully create your user without the second page of user details, and with the correct details on the account creation form
- Edit your profile information and make sure the updates succeed (can be accessed via the dropdown menu in the header)

#### Any background context you want to provide?
- I tried to be sure to leave in code that we're likely to use down the road, and get rid of other obsolete code that will be very easy to copy over from xpro (e.g.: entire component files). Ultimately my choices may look slightly arbitrary to other eyes 😁 

#### Screenshots (if appropriate)
![ss 2021-08-02 at 16 32 50 ](https://user-images.githubusercontent.com/14932219/127926643-68e4029c-6023-4758-9f93-c8a98d19f820.png)
![ss 2021-08-02 at 16 27 57 ](https://user-images.githubusercontent.com/14932219/127927223-503482ea-f967-476d-92fa-44864f684f72.png)
![ss 2021-08-02 at 16 30 49 ](https://user-images.githubusercontent.com/14932219/127926668-5f7d33f7-b1b9-4334-a1e7-3d6309502aa5.png)
